[2024-12-18T10:42:51.276+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-18T10:42:51.284+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2024-12-16T13:30:00+00:00 [queued]>
[2024-12-18T10:42:51.286+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2024-12-16T13:30:00+00:00 [queued]>
[2024-12-18T10:42:51.287+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-18T10:42:51.291+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): concat_data> on 2024-12-16 13:30:00+00:00
[2024-12-18T10:42:51.296+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=95) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-18T10:42:51.296+0000] {standard_task_runner.py:64} INFO - Started process 102 to run task
[2024-12-18T10:42:51.297+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'concat_data', 'scheduled__2024-12-16T13:30:00+00:00', '--job-id', '1433', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmp4cown9ve']
[2024-12-18T10:42:51.298+0000] {standard_task_runner.py:91} INFO - Job 1433: Subtask concat_data
[2024-12-18T10:42:51.313+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2024-12-16T13:30:00+00:00 [running]> on host b7aba691e1ff
[2024-12-18T10:42:51.343+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='concat_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T13:30:00+00:00'
[2024-12-18T10:42:51.343+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-18T10:42:51.346+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2024-12-18T10:42:51.367+0000] {logging_mixin.py:188} INFO -                OPEN     HIGH      LOW    CLOSE      VOLUME
DATE                                                      
2020-10-02  10059.0  10553.0  10059.0  10328.0  23509500.0
2020-10-05  10328.0  10597.0  10283.0  10597.0  13026585.0
2020-10-06  10597.0  10867.0  10597.0  10777.0  10970227.0
2020-10-07  10777.0  10822.0  10463.0  10508.0  17712190.0
2020-10-08  10418.0  10642.0  10328.0  10418.0  14090748.0
...             ...      ...      ...      ...         ...
12/10/2024  25500.0  25500.0  25350.0  25350.0   6655538.0
12/11/2024  25350.0  25500.0  25200.0  25400.0   8847389.0
12/14/2024  25300.0  25300.0  25100.0  25150.0  10650030.0
12/15/2024  25300.0  25300.0  25100.0  25150.0  10650030.0
12/17/2024  25250.0  25250.0  25050.0  25200.0   5964239.0

[1013 rows x 5 columns]
[2024-12-18T10:42:51.452+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-18T10:42:51.452+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-18T10:42:51.455+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=concat_data, run_id=scheduled__2024-12-16T13:30:00+00:00, execution_date=20241216T133000, start_date=20241218T104251, end_date=20241218T104251
[2024-12-18T10:42:51.476+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-12-18T10:42:51.486+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-18T10:42:51.486+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
